psvpfsparser/PfsFilesystem.cpp

```diff
@@ -122,7 +122,7 @@ int PfsFilesystem::decrypt_files(boost::filesystem::path destTitleIdPath) const
       //copy unencrypted files
       else if(is_unencrypted(file->file.m_info.header.type))
       {
-         if(!filepath.copy_existing_file(m_titleIdPath, destTitleIdPath))
+         if(!filepath.copy_existing_file(m_titleIdPath, destTitleIdPath, file->file.m_info.header.size))
```

Comment:
this fix should apply both to icv and unicv?

Reply:
As far as I can tell, unicv pfs files will always have the exact size instead of being padded.

Reply:
Ok. I was just curious if copy_existing_file overload could cause some side effects because decrypt_files method is called for both icv and unicv files. I guess not.

Reply:
sorry misunderstood what you meant earlier. But yeah I think it is fine like this.